GWAS/liftover.ipynb

```diff
@@ -255,11 +255,11 @@
     "    import os.path\n",
     "    input_path=${_input[0]:r}\n",
     "    output_path=${_output[0]:r}\n",
-    "    fr = ${fr}\n",
-    "    to = ${to}\n",
+    "    fr = f'${fr}'\n",
```

Comment:
Please remove `f`. There is no need for that I believe.

Reply:
To clarify, you are saying this does not work? I don't think so. Please double check

```
fr = "${fr}"
```

(without `f` but with quote on the string)